packages/table/src/layers/focusCell.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+export interface IFocusedCellCoordinates {
```

Comment:
Agreed. Just `ICellCoordinate` would work.

This could also be just a tuple of numbers. like `type ICellCoordinate = [number, number]` but it might be easy to forget the order [row, col] vs [col, row].